python/cudf/cudf/utils/ioutils.py

```diff
@@ -215,6 +215,10 @@
     will be written to different directories, but all files will
     have this name.  If nothing is specified, a random uuid4 hex string
     will be used for each file.
+row_group_cols : list, optional, default None
+    Column names by which to partition the dataset across row groups in the
+    resulting Parquet file
+    Columns are partitioned in the order they are given
```

Comment:
Sentences in docstrings should end in a period. Line breaks should be avoided except to soft-wrap.
```suggestion
    Column names by which to partition the dataset across row groups in the
    resulting Parquet file. Columns are partitioned in the order they are
    given.
```